app/src/pages/inside/launchesPage/launchSuiteGrid/launchSuiteGrid.jsx

```diff
@@ -0,0 +1,241 @@
+    <AbsRelTime startTime={rest.value.start_time} user={{ userId: 'artem' }} />
+  </div>
+);
+StartTimeColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+};
+
+const TotalColumn = ({ className, ...rest }) => (
+  <div className={cx('total-col', className)}>
+    <ExecutionStatistics title={rest.title} value={rest.value.statistics.executions.total} bold />
+  </div>
+);
+TotalColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+};
+
+const PassedColumn = ({ className, ...rest }) => (
+  <div className={cx('passed-col', className)}>
+    <ExecutionStatistics title={rest.title} value={rest.value.statistics.executions.passed} />
+  </div>
+);
+PassedColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+};
+
+const FailedColumn = ({ className, ...rest }) => (
+  <div className={cx('failed-col', className)}>
+    <ExecutionStatistics title={rest.title} value={rest.value.statistics.executions.failed} />
+  </div>
+);
+FailedColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+};
+
+const SkippedColumn = ({ className, ...rest }) => (
+  <div className={cx('skipped-col', className)}>
+    <ExecutionStatistics title={rest.title} value={rest.value.statistics.executions.skipped} />
+  </div>
+);
+SkippedColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+};
+
+const PbColumn = ({ className, ...rest }) => (
+  <div className={cx('pb-col', className)}>
+    <DefectStatistics
+      type={'pb'}
+      customProps={rest.customProps}
+      value={rest.value.statistics.defects.product_bug}
+    />
+  </div>
+);
+PbColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+};
+
+const AbColumn = ({ className, ...rest }) => (
+  <div className={cx('ab-col', className)}>
+    <DefectStatistics
+      type={'ab'}
+      customProps={rest.customProps}
+      value={rest.value.statistics.defects.automation_bug}
+    />
+  </div>
+);
+AbColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+};
+
+const SiColumn = ({ className, ...rest }) => (
+  <div className={cx('si-col', className)}>
+    <DefectStatistics
+      type={'si'}
+      customProps={rest.customProps}
+      value={rest.value.statistics.defects.system_issue}
+    />
+  </div>
+);
+SiColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+};
+
+const TiColumn = ({ className, ...rest }) => (
+  <div className={cx('ti-col', className)}>
+    <ToInvestigteStatistics
+      customProps={rest.customProps}
+      value={rest.value.statistics.defects.to_investigate}
+    />
+  </div>
+);
+TiColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+};
+
+const SelectColumn = ({ className }) => (
+  <div className={cx('select-col', className)}>
+    <InputCheckbox />
+  </div>
+);
+SelectColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+};
+
+const COLUMNS = [
+  {
+    component: HamburgerColumn,
+  },
+  {
+    title: {
+      full: 'name',
+      short: 'name',
```

Comment:
It's not necessary to duplicate titles: if `short` is not defined, `full` will be shown all the time.